gazelle/internal/swift/code_dir.go

```diff
@@ -40,3 +40,9 @@ func CodeDirForLocalPackage(pkgDir string, localPkgPath string) string {
 	// Return the local path
 	return filepath.Clean(path)
 }
+
+func CodeDirForRegistryPackage(buildDir string, registryIdentity string, version string) string {
```

Comment:
nit: add a doc comment describing what this does with an example mapping of registry id to path

Reply:
done!